POC/db/models/stock_models/db_models.py

```diff
@@ -106,15 +102,43 @@ class FieldInfoForm(SQLModel):
 class FieldInfo(FieldInfoForm):
     id: int = Field(default=None, primary_key=True)
     db_id: int = Field(default=None)
-    created_at: dt = Field(default=dt.now())
-    updated_at: dt = Field(default=dt.now())
+    created_at: dt = Field(default_factory=dt.now)
+    updated_at: dt = Field(default_factory=dt.now)
     is_active: bool = Field(default=True)
 
+    @field_serializer("created_at", "updated_at", when_used="always")
+    def dt_to_json(self, v: dt, info: FieldSerializationInfo) -> str | dt:
+        return v.replace(microsecond=0)
+
 
 class FieldInfoModel(FieldInfo, table=True):  # type: ignore
     ...
 
 
+class CreateTagsForm(SQLModel):
+    tag_name: str = Field(
+        title="Tag Name",
+        description="The common name for your tag",
+    )
+
+
+class TagsInfo(CreateTagsForm, table=True):  # type: ignore
+    id: Optional[int] = Field(default=None, primary_key=True)
+
+
+class SelectTagForm(SQLModel):
+    tag_name: list[str] = PydanticField(
+        json_schema_extra={"search_url": "/forms/tags/search"}
+    )
+
+    @field_validator("tag_name", mode="before")
```

Comment:
this is a form for selection on or more tags.
validates that no matter what the users response is the selected tags or tag is a list